share/commands/server-upgrade-techniques

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+# @description upgrade techniques in the configuration repository from the packaged ones
+# @man This command will replace the techniques in /var/rudder/configuration-repository/techniques
+# @man by the techniques found in /opt/rudder/share/techniques which is installed by rudder-technique package.
+# @man The upgrade can take care of used defined changes if a proper tag has been set during previous upgrade.
+# @man +
+# @man *Options*:
+# @man +
+# @man *-o*: override existing technique without looking for local changes
+# @man +
+# @man *-t*: do not upgrade, but tag the repository to make next upgrade easier
+# @man +
+# @man *-c*: take the given commit id as a starting point for merging
+# @man +
+
+. "${BASEDIR}/../lib/common.sh"
+. "${BASEDIR}/../lib/api_call.sh"
+
+DISPLAY_COMMAND=false
+
+while getopts "otc:" opt; do
+  case $opt in
+    o)
+      OVERRIDE=true
```

Comment:
Don't you want to set OVERRIDE and TAG to false as default ?  (would put them few lines above above with DISPLAY_COMMAND)

Reply:
It is not necessary since they are compared to true, but it would make this more readable